src/components/DatePickerRange.react.js

```diff
@@ -0,0 +1,306 @@
+                if (setProps) {
+                  setProps({
+                    startDate: startDateStr
+                  });
+                }
+                if (fireEvent) {
+                  fireEvent('change');
+                }
+              }
+              this.props.start_date = startDate;
+              if (endDate != null) {
+                const endDateStr = endDate.format('YYYY-MM-DD');
+                if (setProps) {
+                  setProps({
+                    endDate: endDateStr
+                  });
+                }
+                if (fireEvent) {
+                  fireEvent('change');
+                }
+              }
+            }}
+            focusedInput={ this.state.focusedInput }
+            onFocusChange={focusedInput => this.setState({ focusedInput })}
+            isOutsideRange={date =>
+              date < this.state.minDateRange || date >= this.state.maxDateRange
+            }
+            showClearDates={ this.props.clearable }
+            disabled={ this.props.disabled }
+            keepOpenOnDateSelect={ this.props.stay_open_on_select }
+            reopenPickerOnClearDates={ this.props.reopen_calendar_on_clear }
+            initialVisibleMonth={() => {
+              if (this.state.startDate != null) {
+                return this.state.startDate
+              } else {
+                return this.state.initialVisibleMonth
+              }
+            }}
+            numberOfMonths={ this.props.number_of_months_shown }
+            withPortal={ this.props.with_portal }
+            withFullScreenPortal={ this.props.with_full_screen_portal }
+            firstDayOfWeek={ this.props.first_day_of_week }
+            minimumNights={ this.props.minimum_nights }
+            enableOutsideDays={ this.props.show_outside_days }
+            monthFormat={ this.props.month_format }
+            displayFormat={ this.props.display_format }
+          />
+        );
+    }
+}
+
+DatePickerRange.propTypes = {
+    id: PropTypes.string,
+
+    /**
+     * Specifies the starting date for the component, best practice is to pass
+     * value via datetime object
+     */
+    start_date: PropTypes.string,
+
+    /**
+     * Specifies the ending date for the component, best practice is to pass
+     * value via datetime object
+     */
+    end_date: PropTypes.string,
+
+    /**
+     * Specifies the lowest selectable date for the component,
+     * best practice is to pass value via datetime object
+     */
+    min_date_range: PropTypes.string,
+
+    /**
+     * Specifies the highest selectable date for the component,
+     * best practice is to pass value via datetime object
+     */
+    max_date_range: PropTypes.string,
+
+    /**
+     * Specifies the date that is initially presented when the user
+     * opens the calendar, best practice is to pass value via datetime
+     * object
+     */
+    initial_visible_month: PropTypes.string,
```

Comment:
Oh, is it because the datepicker only shows a single month at a time? But instead of passing in a single month integer, you pass in an entire date string. Is that how it works? If so, I guess that makes sense. I would sort of expect this attribute to take an integer from 1 to 12, but I guess a datestring will be just as easy.